src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -7,8 +7,7 @@
   -->
 
   <PropertyGroup>
-    <Version Condition="'$(StableVersion)' != ''">$(StableVersion)</Version>
-
+    <PackageVersion Condition="'$(PackageVersion)' == '' AND '$(Version)' != ''">$(Version)</PackageVersion>
```

Comment:
>'$(Version)' != '' [](start = 61, length = 18)

I suppose this would be the case when an individual pkgproj wants to force a version? If true, why not just set PackageVersion in the pkgproj instead?